index.js

```diff
@@ -689,20 +719,17 @@ const createJavaBom = async (
           var sbtArgs = [];
           var pluginFile = null;
           if (standalonePluginFile) {
-            sbtArgs = [
-              `-addPluginSbtFile=${tempSbtPlugins}`,
-              `dependencyList::toFile "${dlFile}" --append`,
-            ];
+            sbtArgs = [`-addPluginSbtFile=${tempSbtPlugins}`,`"dependencyList::toFile ${dlFile} --append"`]
           } else {
             // write to the existing plugins file
-            sbtArgs = [`dependencyList::toFile "${dlFile}" --append`];
+            sbtArgs = [`"dependencyList::toFile ${dlFile} --append"`]
             pluginFile = utils.addPlugin(basePath, sbtPluginDefinition);
           }
-          const result = spawnSync(SBT_CMD, sbtArgs, {
-            cwd: basePath,
-            encoding: "utf-8",
-            timeout: TIMEOUT_MS,
-          });
+          const result = spawnSync(
+            SBT_CMD,
+            sbtArgs,
+            { cwd: basePath, shell: true, encoding: "utf-8", timeout: TIMEOUT_MS }
```

Comment:
do we need shell: true ?

Reply:
Yes

Reply:
Added some documentation to make sure this is not removed in the future